### PR TITLE
#10: Colorized console logger, add seconds to time string (closes #10)

### LINKED ISSUE #10: Colorized console logger, add seconds to time string

### PR DESCRIPTION
**Pull request by utaal**
Issue #10

### CODE REVIEW

File: logging/src/main/scala/io.buildo.ingredients/logging/transport/ConsoleLogger.scala [hunk 1]
```diff
@@ -20,7 +22,11 @@ class Console extends Transport {
       fileName <- msg.fileName
       line <- msg.line
     } yield s" [${fileName}:${line}]").getOrElse("")
-    println(s"[${msg.level}] [$timeString] [$name]${fileLineSegment} ${msg.message}")
+    if (colorized) {
+      println(s"[${C.BLUE}${msg.level}${C.RESET}] [${C.WHITE}$timeString${C.RESET}] [${C.YELLOW}$name${C.RESET}]${fileLineSegment} ${msg.message}")
+    } else {
+      println(s"[${msg.level}] [$timeString] [$name]${fileLineSegment} ${msg.message}")
+    }
```
Comment (gabro) [inline logging/src/main/scala/io.buildo.ingredients/logging/transport/ConsoleLogger.scala]: copy pasted template string? Ugh...

Comment (utaal) [inline logging/src/main/scala/io.buildo.ingredients/logging/transport/ConsoleLogger.scala]: oh come on

**Comment by utaal** [comment]
Don't merge!

**Comment by utaal** [comment]
Please merge. @gabro

**Pull request merged**